src/address.cpp

```diff
@@ -44,17 +44,18 @@
 #include <string>
 #include <sstream>
 
-zmq::address_t::address_t (const std::string &protocol_,
```

Comment:
why was this change necessary?

Reply:
I undid the change and it still compiled, so it wasn't necessary.